ivy/functional/backends/paddle/activations.py

```diff
@@ -210,3 +210,27 @@ def hardswish(
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
     return F.hardswish(x)
+
+
+def segment_max(
+    data: paddle.Tensor,
+    segment_ids: paddle.Tensor,
+    /,
+    *,
+    out: Optional[paddle.Tensor] = None,
+) -> paddle.Tensor:
+    unique_segment_ids, segment_counts = paddle.unique(segment_ids)
```

Comment:
have you tried using `paddle.incubate.segment_max` instead?

Reply:
Yes i have simple returned the function with correct arguments